### ISSUE TITLE
[REQUEST] UBC Support

### ISSUE THREAD

**Issue by bookerthegeek** [ForestryMC/ForestryMC#1611]
Hello!

I am requesting that you register your ores with the [UBC](https://github.com/Zeno410/UndergroundBiomesConstructs1.10) if it is installed.

The API can be found [HERE](https://github.com/Zeno410/UndergroundBiomesConstructs1.10/blob/master/src/main/java/exterminatorjeff/undergroundbiomes/api/common/UBOresRegistry.java).

Thank you for considering my request.

**Comment by mezz** [comment]
This was added for a short time in unstable builds, but then removed and put into another mod: https://github.com/mallrat208/UBC-Ore-Registrar/
See this issue for more information:
https://github.com/ForestryMC/ForestryMC/issues/441

**Comment by bookerthegeek** [comment]
@mezz Thank you for that mod listing. 👍 

However if you noticed I linked you to the 1.10 version of the API, and not the 1.7.10 version? The mod that you so kindly provided a linked is for 1.7.10 and as that is EOL I would not have requested a feature to be added to it. (I personally hate wasting mod creators time) 

If that mod exists was for 1.10 I would have used that mod as it does what I am asking for Forestry to do natively.

Edit:
p.s. I do understand their was a [mod](https://github.com/mallrat208/UBC-Ore-Registrar/issues/22) that added all of this compatibility by itself, sadly it is not updated from 1.7.10, although I did post there inquiring about an update.
